opennms-model/src/main/java/org/opennms/netmgt/model/ResourcePath.java

```diff
@@ -141,6 +141,17 @@ public int relativeDepth(ResourcePath child) {
         return numChildEls - numParentEls;
     }
 
+    public ResourcePath getParent() {
+        if(!hasParent()){
+            throw new UnsupportedOperationException("I am on the root level already");
+        }
+        return new ResourcePath(this.m_elements.subList(0, this.m_elements.size()-1));
+    }
+
+    public boolean hasParent() {
+        return this.m_elements.size()>1;
```

Comment:
`this.m_`

Reply:
done. (this changes a opennms model class)